File: Servidor/databaseJavaRest/src/main/java/model/GenericResponse.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

/**
 *
 * @author Gato
 */
public class GenericResponse {
    public long code;
    public String description;

    public GenericResponse() {
    }

    public GenericResponse(long codeError, String descripError) {
        this.code = codeError;
        this.description = descripError;
    }

    public long getCodeError() {
        return code;
    }

    public void setCodeError(long codeError) {
        this.code = codeError;
    }

    public String getDescripError() {
        return description;
    }

    public void setDescripError(String descripError) {
        this.description = descripError;
    }
    
    
}

File: Servidor/databasePHPClient/src/model/GenericResponse.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

namespace model;

/**
 * Description of GenericResponse
 *
 * @author Gato
 */
class GenericResponse {

    public $code;
    public $description;

    public function __construct($code, $description) {
        $this->code = $code;
        $this->description = $description;
    }

    public function getCode() {
        return $this->code;
    }

    public function getDescription() {
        return $this->description;
    }

    public function setCode($code) {
        $this->code = $code;
    }

    public function setDescription($description) {
        $this->description = $description;
    }

}
